### PR TITLE
DM-39174: Update for ts_tcpip v1.1

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

**Review by wvreeven — COMMENTED** [review]
A few comments for you to look at. I will have a look later again.

File: python/lsst/ts/atdome/mock_controller.py [hunk 1]
```diff
@@ -167,73 +163,58 @@ def __init__(
...
+        except Exception as e:
+            print(f"read loop failed: {e!r}")
 
-        server = self.server
-        self.server = None
-        server.close()
-        await asyncio.wait_for(server.wait_closed(), timeout=5)
+    async def write_prompt(self):
+        """Write an unterminated ">" prompt."""
+        await self.write(">".encode("utf-8"))
```
[the remark this answers is not in the document]
Comment (r-owen) [reply python/lsst/ts/atdome/mock_controller.py:179]: I do agree that a terminator argument for write_str might be useful. But this is such a corner case that I'm inclined to just live with it.

File: doc/version_history.rst [hunk 2]
```diff
@@ -6,6 +6,22 @@
...
+v1.11.0
+-------
+
+* Update to use ts_tcpip.
+* Update to use ts_pre_commit_conf.
+* ``Jenkinsfile``: use new shared library.
+
+Requires:
+
+* ts_salobj 6.3
```
[the remark this answers is not in the document]
Comment (r-owen) [reply doc/version_history.rst:18]: I honestly don't know. I suppose I may as well say 7. It's what we'd be using in any case.

**Comment by r-owen** [comment]
Great start. I have addressed all your concerns (though the last one only with a reply, no changes to the code).

**Review by wvreeven — APPROVED** [review]
Looks good to me.

**Pull request merged**